src/backends/backend/triton_model_instance.cc

```diff
@@ -131,9 +157,69 @@ TritonModelInstance::CreateInstance(
 }
 
 Status
+TritonModelInstance::SetBackendThread()
+{
+  // TODO: Currently each instance has a dedicated TritonBackendThread.
+  // For device blocking execution policy we must share the
+  // TritonBackendThread object with the instances on same device.
+  // FIXME: Get the nice value for the thread from config
```

Comment:
We can remove the "nice" setting that we are currently using for prioritization as it is not really effective and we now have the rate limiter for managing prioritization.